app/Controllers/userController.php

```diff
@@ -76,10 +76,14 @@ public function updateAction() {
 
 			$apiPasswordPlain = Minz_Request::param('apiPasswordPlain', '', true);
 
+			$ok = false;
+
 			$username = Minz_Request::param('username');
-			$ok = self::updateUser($username, $passwordPlain, $apiPasswordPlain, array(
+			if (!empty($username)) {
```

Comment:
Would the check make more sense at the top of updateUser() itself? Perhaps similar issues could occur elsewhere.

```php
if(empty($user) {
  return false;
}
```

Reply:
But then the user won't get the visible feedback (see below for the `Minz_Request::bad()` call).
Depends on the wanted behavior in a case where the user submits an empty form.

Reply:
I said `updateUser()`, ergo line 48. ;-)

(Not saying it's the way to go, but I definitely wasn't suggesting to avoid feedback.)

Reply:
Right, misunderstood your suggestion. Thats better, as the function is used from multiple places.